src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,42 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        Driver petrenkoPP = new Driver(1L, "Petrenko P.P.", "KIA000001");
+        Driver ivanenkoII = new Driver(2L, "Ivanenko I.I.", "KIB000001");
```

Comment:
id will be generated in DB automatically. its redundant parameter in the constructor